binr/rules.mk

```diff
@@ -52,6 +52,8 @@ endif
 endif
 endif
 
+LDFLAGS+=-llz4
```

Comment:
nope. use include $(SHLD)/lz4/deps.mk

Reply:
also, we want static linking only and it is better to specify the full path to the archive rather than linking with -l, which will use the system library if found before the archive path